src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/InterceptingPropertyValueProviderBase.cs

```diff
@@ -27,13 +25,13 @@ public virtual Task<string> OnGetUnevaluatedPropertyValueAsync(string propertyNa
 
         public virtual async Task<bool> IsValueDefinedInContextAsync(string propertyName, IProjectProperties defaultProperties)
         {
+            var t = await defaultProperties.IsValueInheritedAsync(propertyName);
             return !await defaultProperties.IsValueInheritedAsync(propertyName);
         }
 
-        private protected static async Task<bool> IsValueDefinedInContextMSBuildPropertiesAsync(IProjectProperties defaultProperties, string[] msBuildPropertyNames)
+        internal static async Task<bool> IsValueDefinedInContextMsBuildPropertiesAsync(IProjectProperties defaultProperties, string[] msBuildPropertyNames)
```

Comment:
Super nit: .NET conventions have two letter acronyms capitalized, while longer ones are not. So this name should be `MSBuild`, not `MsBuild`.

This allows you to read something like "MSUI" as two words, while "MSSoapXml" is clearly three words.